deploy/helm/grafana-operator/values.yaml

```diff
@@ -77,12 +86,17 @@ podAnnotations: {}
 # -- pod security context
 podSecurityContext: {}
 
-# -- grafana operator container security context
+# https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
+  # -- grafana operator does not require default capabilities
```

Comment:
In my view, when comments serve as documentation, we should not write what we actually configured, we should rather say what an option allows to set (just try to read `README.md` instead of values, so you'll see what I'm referring to).
In this case, it could something along the lines of `# -- A list of capabilities to drop`. The fact that the operator does not need any capabilities is highlighted through the default value.